lesson7-questions.txt

```diff
@@ -75,6 +100,11 @@ forum?  Hint: You need to have a where in your statement. Try this out in the co
 You will see the SQL that is executed logged to the screen.  What is that SQL?
 
 
+user_posts_in_forum = Post.where(user_id: user.id, forum_id: forum.id)
```

Comment:
This works. An alternative solution could make use of the association methods, like so:

```ruby
user.posts.where(forum: forum)
```